src/fillTank.test.js

```diff
@@ -1,11 +1,111 @@
 'use strict';
 
 describe('fillTank', () => {
-  // const { fillTank } = require('./fillTank');
+  const { fillTank } = require('./fillTank');
 
   it('should ', () => {
 
   });
 
-  // write tests here
+  it(`should not change fuelRemains if user has no money`, () => {
+    const customer1 = {
+      money: 0,
+      vehicle: {
+        maxTankCapacity: 40, 
+        fuelRemains: 0, 
+      },
+    };
+
+    fillTank(customer1, 10, 5);
+
+    expect(customer1.vehicle.fuelRemains).toBe(0);
+  });
+
+  it(`User.money should not change more`
+    + ` than the cost of filling up to maxTankCapacity`, () => {
+    const customer1 = {
+      money: 50,
+      vehicle: {
+        maxTankCapacity: 40,
+        fuelRemains: 20,
+      },
+    };
+
+    fillTank(customer1, 1, 40);
+
+    expect(customer1.money).toBe(30);
+  });
+
+  it(`should change fuelRemains to maxTankCapacity`, () => {
+    const customer1 = {
+      money: 100,
+      vehicle: {
+        maxTankCapacity: 40,
+        fuelRemains: 0,
+      },
+    };
+
+    fillTank(customer1, 1);
```

Comment:
In the test case `should change fuelRemains to maxTankCapacity`, the `fillTank` function is called with only two arguments. According to the task description, the function should handle this by assuming a full tank is ordered. Ensure that this behavior is correctly implemented in the `fillTank` function.